cmd/juju/subnet/list.go

```diff
@@ -47,14 +47,29 @@ output formats include "yaml" (default) and "json". To redirect the
 output to a file, use --output.
 `
 
+const listCommandExample = `
+To list all subnets known to Juju, use:
+
+    juju subnets
+
+To list subnets associated with a specific network space, use the --space
+
+    juju subnets --space my-space
+
+To list subnets associated with a specific availability zone, use the --zone
+
+    juju subnets --zone my-zone
+`
+
```

Comment:
For consistency with what we have everywhere else, I'd cut out the ", use..." bit. So:

> To list all subnets known to Juju:

> To list subnets associated with a specific network space:

> To list subnets associated with a specific availability zone: